src/main/java/com/cs56fitnessapp/models/Day.java

```diff
@@ -1,13 +1,33 @@
 package com.cs56fitnessapp.models;
 
 /**
- * @author Ksenia Koldaeva
- * Created: 9/17/17
- * Last Updated: 10/1/17
+ * @author Saori Shigehisa
+ * Created: 10/6/17
+ * Last Updated: 10/7/17
  */
+import java.util.ArrayList;
+import java.util.Date;
 
 public class Day implements ActivityFacts, DietFacts {
 
+    /** Date */
+    private Date date;
```

Comment:
You did the right implementation according to the UML. 
My bad - we're moving from `Date` to modern Java class `LocalDate`. I just modified the UML. So will need to change it to the `LocalDate` type.

Reply:
fixed it.